cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java

```diff
@@ -136,6 +136,25 @@ public static int getSequenceId(String name) {
   }
 
   /**
+   * Returns the writer instance number of the given file name.
+   *
+   * @param name Name of the file.
+   * @return The writer instance number of the stream file.
+   * @see StreamInputFormat
+   */
+  public static int getWriterInstanceId(String name) {
```

Comment:
I think this method shouldn't be in `StreamUtils` because instance id is not part of stream file name pattern. The standard pattern is `prefix.seqId.ext` only.